### DISCUSSION TITLE
restore hangs on failing source

### DISCUSSION THREAD

**Discussion by umerkle** [NuGet/Home#13136] [Q&A]
Hi there,

i have configured multiple (private and public) nuget sources to be used for my projects.
They used to work fine, until one of them was (temporary) unavailable.
When i tried to build a new project for the first time with `dotnet build`, it noticed to one failing source with error NU1301 

> 'The service index for source "xxx" could not be loaded.'
 
It tried again and again and again, but never came to an end. In the current project, the failing sources werent needed to restore the project. So in my humble opinion, i would think it could just stop trying to get this index for failed sources and just go on with the working ones. If restore could be done OK with just them, why bother with failing sources?

And also: when i tried to stop the rebuild with <Ctrl>-<C>, it said 

> trying to cancel the build.

and in yellow also

> warning MSB4220: Waiting for cancelling the currently running task "RestoreTask"

But also, never stopped working. I had kill the dotnet processes to stop it.

FYI: I'm working in Linux with dotnet 8.

**Comment by theDRB123** [comment]
getting the same problem with building a project, have you found a solution to this ?

**Reply by umerkle** [reply]
The problem is gone now. But unfortunately I don't know why...
